repair_quotation_manual_sync/models/stock_move.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2025 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    invoiceable = fields.Boolean(
+        help="Check this box if the product is billable. "
+        "Only invoiceable products will be included in the quotation.",
+        default=True,
+    )
+
+    def _create_repair_sale_order_line(self):
+        invoiceable_moves = self.filtered("invoiceable")
+        if invoiceable_moves:
+            super(StockMove, invoiceable_moves)._create_repair_sale_order_line()
+        return None
```

Comment:
```suggestion
        return super(StockMove, self.filtered("invoiceable"))._create_repair_sale_order_line()
```
Here, you must take into account the scenario for which we add a new line in the RO with the `invoiceable` field marked, it will skip the manual sync logics. Then, in order to do so, you should consider adding a context in the call of this method in the `action_sync_repair_lines`, something like `repair_lines_manual_sync` to be used here and only create lines for those that come from this context.